portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
+          <li><a href="https://www.instagram.com/kvngsno">Instagram</a></li>
+          <li><a href="https://www.linkedin.com/in/jalen-jenkins-b75772186">Linkedin</a><li> 
+          </ul>
+
+    <div id="image"> <img src = "images/self image.jpg" alt = "A picture of myself" style="width:128px;height:128px;"></div>
+    
+    <div id="content"> 
+
+      <h1>Jalen's Portfolio</h1>
+      
+      <p>Welcome to my Portfolio!</p>
+      
+      <h2>About me</h2>
+      <p> 
+          <br> My name is Jalen Jenkins and I am currently a Computer Science Major in Cornell University's Class
```

Comment:
I'd remove this `br` tag - if you're looking to get more space between the h2 and p, consider adding extra margin in CSS, either to the bottom of h2, or the top of p.